website/docs/language/settings/backends/azurerm.html.md

```diff
@@ -11,7 +11,7 @@ description: |-
 
 **Kind: Standard (with state locking)**
 
-Stores the state as a Blob with the given Key within the Blob Container within [the Blob Storage Account](https://docs.microsoft.com/azure/storage/common/storage-introduction). This backend also supports state locking and consistency checking via native capabilities of Azure Blob Storage.
+Stores the state as a Blob with the given Key within the Blob Container within [the Blob Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction). This backend also supports state locking and consistency checking via native capabilities of Azure Blob Storage.
```

Comment:
if we remove this it'll redirect to the users locale:

```suggestion
Stores the state as a Blob with the given Key within the Blob Container within [the Blob Storage Account](https://docs.microsoft.com/azure/storage/common/storage-introduction). This backend also supports state locking and consistency checking via native capabilities of Azure Blob Storage.
```

Reply:
This one breaks the website-test as a link that has permanently moved. I think we'll need to keep it this way